python/shotgun_desktop/startup.py

```diff
@@ -64,23 +64,32 @@ def add_to_python_path(bundled_path, env_var_override, module_name):
                                     InvalidPipelineConfiguration)
 
 
-def __is_64bit_python():
+global_debug_flag_at_startup = None
+
+
```

Comment:
git does a really poor job at comparing files here. It's simpler to look at [the new code](https://github.com/shotgunsoftware/tk-framework-desktopstartup/blob/d47def0f394c4d7b69807e63c4d8054c6c8761f9/python/shotgun_desktop/startup.py#L67-L85) without the diff.